src/tools/clippy/tests/ui/wildcard_enum_match_arm.fixed

```diff
@@ -7,7 +7,8 @@
     dead_code,
     clippy::single_match,
     clippy::wildcard_in_or_patterns,
-    clippy::unnested_or_patterns
+    clippy::unnested_or_patterns,
+    clippy::diverging_sub_expression
```

Comment:
With this change you have to adapt the `stderr` file, because every error moved one line. You could either +1 every line number in the `stderr` file or change this to
```suggestion
    clippy::unnested_or_patterns, clippy::diverging_sub_expression
```

and we'll fix the formatting during sync.

Reply:
I just squeezed it onto the same line so that we can get this merged :)